## `nixpkgs-review` result for [#514334](https://github.com/NixOS/nixpkgs/pull/514334)

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 514334`
Commit: [`0dc33800f1a146d5c5423989bcbd750f2280ac96`](https://github.com/NixOS/nixpkgs/commit/0dc33800f1a146d5c5423989bcbd750f2280ac96) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/0dc33800f1a146d5c5423989bcbd750f2280ac96..pull/514334/head))
Merge: [`1e4c42b501584485355eda3a03de369fa9bd294e`](https://github.com/NixOS/nixpkgs/commit/1e4c42b501584485355eda3a03de369fa9bd294e)

Logs: https://github.com/eljamm/nixpkgs-review-gha/actions/runs/25039488217


---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 6 packages marked as broken and skipped:</summary>
  <ul>
    <li>linuxKernel.packages.linux_xanmod.broadcom_sta</li>
    <li>linuxKernel.packages.linux_xanmod_latest.broadcom_sta</li>
    <li>linuxKernel.packages.linux_xanmod_stable.broadcom_sta</li>
    <li>linuxPackages_xanmod.broadcom_sta</li>
    <li>linuxPackages_xanmod_latest.broadcom_sta</li>
    <li>linuxPackages_xanmod_stable.broadcom_sta</li>
  </ul>
</details>
<details>
  <summary>:x: 108 packages failed to build:</summary>
  <ul>
    <li>linux_xanmod_latest (linuxPackages_xanmod_latest.kernel, linux_xanmod_stable)</li>
    <li>linux_xanmod_latest.dev (linuxPackages_xanmod_latest.kernel.dev, linux_xanmod_stable.dev)</li>
    <li>linux_xanmod_latest.modules (linuxPackages_xanmod_latest.kernel.modules, linux_xanmod_stable.modules)</li>
    <li>linuxPackages_xanmod.lttng-modules (linuxKernel.packages.linux_xanmod.lttng-modules)</li>
    <li>linuxPackages_xanmod.mbp2018-bridge-drv (linuxKernel.packages.linux_xanmod.mbp2018-bridge-drv)</li>
    <li>linuxPackages_xanmod.vmm_clock (linuxKernel.packages.linux_xanmod.vmm_clock)</li>
    <li>linuxPackages_xanmod_latest.acer-wmi-battery (linuxKernel.packages.linux_xanmod_latest.acer-wmi-battery, linuxPackages_xanmod_stable.acer-wmi-battery)</li>
    <li>linuxPackages_xanmod_latest.acpi_call (linuxKernel.packages.linux_xanmod_latest.acpi_call, linuxPackages_xanmod_stable.acpi_call)</li>
    <li>linuxPackages_xanmod_latest.ajantv2 (linuxKernel.packages.linux_xanmod_latest.ajantv2, linuxPackages_xanmod_stable.ajantv2)</li>
    <li>linuxPackages_xanmod_latest.akvcam (linuxKernel.packages.linux_xanmod_latest.akvcam, linuxPackages_xanmod_stable.akvcam)</li>
    <li>linuxPackages_xanmod_latest.amdgpu-i2c (linuxKernel.packages.linux_xanmod_latest.amdgpu-i2c, linuxPackages_xanmod_stable.amdgpu-i2c)</li>
    <li>linuxPackages_xanmod_latest.amneziawg (linuxKernel.packages.linux_xanmod_latest.amneziawg, linuxPackages_xanmod_stable.amneziawg)</li>
    <li>linuxPackages_xanmod_latest.apfs (linuxKernel.packages.linux_xanmod_latest.apfs, linuxPackages_xanmod_stable.apfs)</li>
    <li>linuxPackages_xanmod_latest.asus-ec-sensors (linuxKernel.packages.linux_xanmod_latest.asus-ec-sensors, linuxPackages_xanmod_stable.asus-ec-sensors)</li>
    <li>linuxPackages_xanmod_latest.batman_adv (linuxKernel.packages.linux_xanmod_latest.batman_adv, linuxPackages_xanmod_stable.batman_adv)</li>
    <li>linuxPackages_xanmod_latest.bbswitch (linuxKernel.packages.linux_xanmod_latest.bbswitch, linuxPackages_xanmod_stable.bbswitch)</li>
    <li>linuxPackages_xanmod_latest.bcachefs (linuxKernel.packages.linux_xanmod_latest.bcachefs, linuxPackages_xanmod_stable.bcachefs)</li>
    <li>linuxPackages_xanmod_latest.ch9344 (linuxKernel.packages.linux_xanmod_latest.ch9344, linuxPackages_xanmod_stable.ch9344)</li>
    <li>linuxPackages_xanmod_latest.chipsec (linuxKernel.packages.linux_xanmod_latest.chipsec, linuxPackages_xanmod_stable.chipsec)</li>
    <li>linuxPackages_xanmod_latest.chipsec.dist (linuxKernel.packages.linux_xanmod_latest.chipsec.dist, linuxPackages_xanmod_stable.chipsec.dist)</li>
    <li>linuxPackages_xanmod_latest.corefreq (linuxKernel.packages.linux_xanmod_latest.corefreq, linuxPackages_xanmod_stable.corefreq)</li>
    <li>linuxPackages_xanmod_latest.cryptodev (linuxKernel.packages.linux_xanmod_latest.cryptodev, linuxPackages_xanmod_stable.cryptodev)</li>
    <li>linuxPackages_xanmod_latest.ddcci-driver (linuxKernel.packages.linux_xanmod_latest.ddcci-driver, linuxPackages_xanmod_stable.ddcci-driver)</li>
    <li>linuxPackages_xanmod_latest.decklink (linuxKernel.packages.linux_xanmod_latest.decklink, linuxPackages_xanmod_stable.decklink)</li>
    <li>linuxPackages_xanmod_latest.digimend (linuxKernel.packages.linux_xanmod_latest.digimend, linuxPackages_xanmod_stable.digimend)</li>
    <li>linuxPackages_xanmod_latest.dpdk-kmods (linuxKernel.packages.linux_xanmod_latest.dpdk-kmods, linuxPackages_xanmod_stable.dpdk-kmods)</li>
    <li>linuxPackages_xanmod_latest.drbd (linuxKernel.packages.linux_xanmod_latest.drbd, linuxPackages_xanmod_stable.drbd)</li>
    <li>linuxPackages_xanmod_latest.ecapture (linuxKernel.packages.linux_xanmod_latest.ecapture, linuxPackages_xanmod_stable.ecapture)</li>
    <li>linuxPackages_xanmod_latest.ena (linuxKernel.packages.linux_xanmod_latest.ena, linuxPackages_xanmod_stable.ena)</li>
    <li>linuxPackages_xanmod_latest.evdi (linuxKernel.packages.linux_xanmod_latest.evdi, linuxPackages_xanmod_stable.evdi)</li>
    <li>linuxPackages_xanmod_latest.facetimehd (linuxKernel.packages.linux_xanmod_latest.facetimehd, linuxPackages_xanmod_stable.facetimehd)</li>
    <li>linuxPackages_xanmod_latest.fanout (linuxKernel.packages.linux_xanmod_latest.fanout, linuxPackages_xanmod_stable.fanout)</li>
    <li>linuxPackages_xanmod_latest.framework-laptop-kmod (linuxKernel.packages.linux_xanmod_latest.framework-laptop-kmod, linuxPackages_xanmod_stable.framework-laptop-kmod)</li>
    <li>linuxPackages_xanmod_latest.fwts-efi-runtime (linuxKernel.packages.linux_xanmod_latest.fwts-efi-runtime, linuxPackages_xanmod_stable.fwts-efi-runtime)</li>
    <li>linuxPackages_xanmod_latest.gasket (linuxKernel.packages.linux_xanmod_latest.gasket, linuxPackages_xanmod_stable.gasket)</li>
    <li>linuxPackages_xanmod_latest.gcadapter-oc-kmod (linuxKernel.packages.linux_xanmod_latest.gcadapter-oc-kmod, linuxPackages_xanmod_stable.gcadapter-oc-kmod)</li>
    <li>linuxPackages_xanmod_latest.hid-fanatecff (linuxKernel.packages.linux_xanmod_latest.hid-fanatecff, linuxPackages_xanmod_stable.hid-fanatecff)</li>
    <li>linuxPackages_xanmod_latest.hid-ite8291r3 (linuxKernel.packages.linux_xanmod_latest.hid-ite8291r3, linuxPackages_xanmod_stable.hid-ite8291r3)</li>
    <li>linuxPackages_xanmod_latest.hid-t150 (linuxKernel.packages.linux_xanmod_latest.hid-t150, linuxPackages_xanmod_stable.hid-t150)</li>
    <li>linuxPackages_xanmod_latest.hid-tmff2 (linuxKernel.packages.linux_xanmod_latest.hid-tmff2, linuxPackages_xanmod_stable.hid-tmff2)</li>
    <li>linuxPackages_xanmod_latest.hpuefi-mod (linuxKernel.packages.linux_xanmod_latest.hpuefi-mod, linuxPackages_xanmod_stable.hpuefi-mod)</li>
    <li>linuxPackages_xanmod_latest.ipu6-drivers (linuxKernel.packages.linux_xanmod_latest.ipu6-drivers, linuxPackages_xanmod_stable.ipu6-drivers)</li>
    <li>linuxPackages_xanmod_latest.it87 (linuxKernel.packages.linux_xanmod_latest.it87, linuxPackages_xanmod_stable.it87)</li>
    <li>linuxPackages_xanmod_latest.jool (linuxKernel.packages.linux_xanmod_latest.jool, linuxPackages_xanmod_stable.jool)</li>
    <li>linuxPackages_xanmod_latest.kvmfr (linuxKernel.packages.linux_xanmod_latest.kvmfr, linuxPackages_xanmod_stable.kvmfr)</li>
    <li>linuxPackages_xanmod_latest.lenovo-legion-module (linuxKernel.packages.linux_xanmod_latest.lenovo-legion-module, linuxPackages_xanmod_stable.lenovo-legion-module)</li>
    <li>linuxPackages_xanmod_latest.linux-gpib (linuxKernel.packages.linux_xanmod_latest.linux-gpib, linuxPackages_xanmod_stable.linux-gpib)</li>
    <li>linuxPackages_xanmod_latest.liquidtux (linuxKernel.packages.linux_xanmod_latest.liquidtux, linuxPackages_xanmod_stable.liquidtux)</li>
    <li>linuxPackages_xanmod_latest.lkrg (linuxKernel.packages.linux_xanmod_latest.lkrg, linuxPackages_xanmod_stable.lkrg)</li>
    <li>linuxPackages_xanmod_latest.lttng-modules (linuxKernel.packages.linux_xanmod_latest.lttng-modules, linuxPackages_xanmod_stable.lttng-modules)</li>
    <li>linuxPackages_xanmod_latest.mbp2018-bridge-drv (linuxKernel.packages.linux_xanmod_latest.mbp2018-bridge-drv, linuxPackages_xanmod_stable.mbp2018-bridge-drv)</li>
    <li>linuxPackages_xanmod_latest.mdio-netlink (linuxKernel.packages.linux_xanmod_latest.mdio-netlink, linuxPackages_xanmod_stable.mdio-netlink)</li>
    <li>linuxPackages_xanmod_latest.msi-ec (linuxKernel.packages.linux_xanmod_latest.msi-ec, linuxPackages_xanmod_stable.msi-ec)</li>
    <li>linuxPackages_xanmod_latest.mstflint_access (linuxKernel.packages.linux_xanmod_latest.mstflint_access, linuxPackages_xanmod_stable.mstflint_access)</li>
    <li>linuxPackages_xanmod_latest.nct6687d (linuxKernel.packages.linux_xanmod_latest.nct6687d, linuxPackages_xanmod_stable.nct6687d)</li>
    <li>linuxPackages_xanmod_latest.netatop (linuxKernel.packages.linux_xanmod_latest.netatop, linuxPackages_xanmod_stable.netatop)</li>
    <li>linuxPackages_xanmod_latest.new-lg4ff (linuxKernel.packages.linux_xanmod_latest.new-lg4ff, linuxPackages_xanmod_stable.new-lg4ff)</li>
    <li>linuxPackages_xanmod_latest.nullfs (linuxKernel.packages.linux_xanmod_latest.nullfs, linuxPackages_xanmod_stable.nullfs)</li>
    <li>linuxPackages_xanmod_latest.nvidia_x11_beta_open (linuxKernel.packages.linux_xanmod_latest.nvidia_x11_beta_open, linuxPackages_xanmod_stable.nvidia_x11_beta_open)</li>
    <li>linuxPackages_xanmod_latest.nvidia_x11_latest_open (linuxKernel.packages.linux_xanmod_latest.nvidia_x11_latest_open, linuxPackages_xanmod_latest.nvidia_x11_production_open, linuxPackages_xanmod_latest.nvidia_x11_stable_open, linuxPackages_xanmod_stable.nvidia_x11_latest_open, linuxPackages_xanmod_stable.nvidia_x11_production_open, linuxPackages_xanmod_stable.nvidia_x11_stable_open)</li>
    <li>linuxPackages_xanmod_latest.nvidia_x11_vulkan_beta_open (linuxKernel.packages.linux_xanmod_latest.nvidia_x11_vulkan_beta_open, linuxPackages_xanmod_stable.nvidia_x11_vulkan_beta_open)</li>
    <li>linuxPackages_xanmod_latest.nxp-pn5xx (linuxKernel.packages.linux_xanmod_latest.nxp-pn5xx, linuxPackages_xanmod_stable.nxp-pn5xx)</li>
    <li>linuxPackages_xanmod_latest.openafs (linuxKernel.packages.linux_xanmod_latest.openafs, linuxPackages_xanmod_latest.openafs_1_8, linuxPackages_xanmod_stable.openafs, linuxPackages_xanmod_stable.openafs_1_8)</li>
    <li>linuxPackages_xanmod_latest.openrazer (linuxKernel.packages.linux_xanmod_latest.openrazer, linuxPackages_xanmod_stable.openrazer)</li>
    <li>linuxPackages_xanmod_latest.opensnitch-ebpf (linuxKernel.packages.linux_xanmod_latest.opensnitch-ebpf, linuxPackages_xanmod_stable.opensnitch-ebpf)</li>
    <li>linuxPackages_xanmod_latest.qc71_laptop (linuxKernel.packages.linux_xanmod_latest.qc71_laptop, linuxPackages_xanmod_stable.qc71_laptop)</li>
    <li>linuxPackages_xanmod_latest.r8125 (linuxKernel.packages.linux_xanmod_latest.r8125, linuxPackages_xanmod_stable.r8125)</li>
    <li>linuxPackages_xanmod_latest.r8168 (linuxKernel.packages.linux_xanmod_latest.r8168, linuxPackages_xanmod_stable.r8168)</li>
    <li>linuxPackages_xanmod_latest.rr-zen_workaround (linuxKernel.packages.linux_xanmod_latest.rr-zen_workaround, linuxPackages_xanmod_stable.rr-zen_workaround)</li>
    <li>linuxPackages_xanmod_latest.rtl8189es (linuxKernel.packages.linux_xanmod_latest.rtl8189es, linuxPackages_xanmod_stable.rtl8189es)</li>
    <li>linuxPackages_xanmod_latest.rtl8189fs (linuxKernel.packages.linux_xanmod_latest.rtl8189fs, linuxPackages_xanmod_stable.rtl8189fs)</li>
    <li>linuxPackages_xanmod_latest.rtl8821ce (linuxKernel.packages.linux_xanmod_latest.rtl8821ce, linuxPackages_xanmod_stable.rtl8821ce)</li>
    <li>linuxPackages_xanmod_latest.rtl8821cu (linuxKernel.packages.linux_xanmod_latest.rtl8821cu, linuxPackages_xanmod_stable.rtl8821cu)</li>
    <li>linuxPackages_xanmod_latest.rtl88x2bu (linuxKernel.packages.linux_xanmod_latest.rtl88x2bu, linuxPackages_xanmod_stable.rtl88x2bu)</li>
    <li>linuxPackages_xanmod_latest.rtw88 (linuxKernel.packages.linux_xanmod_latest.rtw88, linuxPackages_xanmod_stable.rtw88)</li>
    <li>linuxPackages_xanmod_latest.rust-out-of-tree-module (linuxKernel.packages.linux_xanmod_latest.rust-out-of-tree-module, linuxPackages_xanmod_stable.rust-out-of-tree-module)</li>
    <li>linuxPackages_xanmod_latest.ryzen-smu (linuxKernel.packages.linux_xanmod_latest.ryzen-smu, linuxPackages_xanmod_stable.ryzen-smu)</li>
    <li>linuxPackages_xanmod_latest.sheep-net (linuxKernel.packages.linux_xanmod_latest.sheep-net, linuxPackages_xanmod_stable.sheep-net)</li>
    <li>linuxPackages_xanmod_latest.shufflecake (linuxKernel.packages.linux_xanmod_latest.shufflecake, linuxPackages_xanmod_stable.shufflecake)</li>
    <li>linuxPackages_xanmod_latest.shufflecake.bin (linuxKernel.packages.linux_xanmod_latest.shufflecake.bin, linuxPackages_xanmod_stable.shufflecake.bin)</li>
    <li>linuxPackages_xanmod_latest.sysdig (linuxKernel.packages.linux_xanmod_latest.sysdig, linuxPackages_xanmod_stable.sysdig)</li>
    <li>linuxPackages_xanmod_latest.system76 (linuxKernel.packages.linux_xanmod_latest.system76, linuxPackages_xanmod_stable.system76)</li>
    <li>linuxPackages_xanmod_latest.system76-acpi (linuxKernel.packages.linux_xanmod_latest.system76-acpi, linuxPackages_xanmod_stable.system76-acpi)</li>
    <li>linuxPackages_xanmod_latest.system76-io (linuxKernel.packages.linux_xanmod_latest.system76-io, linuxPackages_xanmod_stable.system76-io)</li>
    <li>linuxPackages_xanmod_latest.systemtap (linuxKernel.packages.linux_xanmod_latest.systemtap, linuxPackages_xanmod_stable.systemtap)</li>
    <li>linuxPackages_xanmod_latest.tp_smapi (linuxKernel.packages.linux_xanmod_latest.tp_smapi, linuxPackages_xanmod_stable.tp_smapi)</li>
    <li>linuxPackages_xanmod_latest.trelay (linuxKernel.packages.linux_xanmod_latest.trelay, linuxPackages_xanmod_stable.trelay)</li>
    <li>linuxPackages_xanmod_latest.tsme-test (linuxKernel.packages.linux_xanmod_latest.tsme-test, linuxPackages_xanmod_stable.tsme-test)</li>
    <li>linuxPackages_xanmod_latest.tt-kmd (linuxKernel.packages.linux_xanmod_latest.tt-kmd, linuxPackages_xanmod_stable.tt-kmd)</li>
    <li>linuxPackages_xanmod_latest.tuxedo-drivers (linuxKernel.packages.linux_xanmod_latest.tuxedo-drivers, linuxPackages_xanmod_stable.tuxedo-drivers)</li>
    <li>linuxPackages_xanmod_latest.universal-pidff (linuxKernel.packages.linux_xanmod_latest.universal-pidff, linuxPackages_xanmod_stable.universal-pidff)</li>
    <li>linuxPackages_xanmod_latest.v4l2loopback (linuxKernel.packages.linux_xanmod_latest.v4l2loopback, linuxPackages_xanmod_stable.v4l2loopback)</li>
    <li>linuxPackages_xanmod_latest.v4l2loopback.bin (linuxKernel.packages.linux_xanmod_latest.v4l2loopback.bin, linuxPackages_xanmod_stable.v4l2loopback.bin)</li>
    <li>linuxPackages_xanmod_latest.v86d (linuxKernel.packages.linux_xanmod_latest.v86d, linuxPackages_xanmod_stable.v86d)</li>
    <li>linuxPackages_xanmod_latest.veikk-linux-driver (linuxKernel.packages.linux_xanmod_latest.veikk-linux-driver, linuxPackages_xanmod_stable.veikk-linux-driver)</li>
    <li>linuxPackages_xanmod_latest.vendor-reset (linuxKernel.packages.linux_xanmod_latest.vendor-reset, linuxPackages_xanmod_stable.vendor-reset)</li>
    <li>linuxPackages_xanmod_latest.vhba (linuxKernel.packages.linux_xanmod_latest.vhba, linuxPackages_xanmod_stable.vhba)</li>
    <li>linuxPackages_xanmod_latest.virtio_vmmci (linuxKernel.packages.linux_xanmod_latest.virtio_vmmci, linuxPackages_xanmod_stable.virtio_vmmci)</li>
    <li>linuxPackages_xanmod_latest.virtualbox (linuxKernel.packages.linux_xanmod_latest.virtualbox, linuxPackages_xanmod_stable.virtualbox)</li>
    <li>linuxPackages_xanmod_latest.virtualboxGuestAdditions (linuxKernel.packages.linux_xanmod_latest.virtualboxGuestAdditions, linuxPackages_xanmod_stable.virtualboxGuestAdditions)</li>
    <li>linuxPackages_xanmod_latest.vmm_clock (linuxKernel.packages.linux_xanmod_latest.vmm_clock, linuxPackages_xanmod_stable.vmm_clock)</li>
    <li>linuxPackages_xanmod_latest.vmware (linuxKernel.packages.linux_xanmod_latest.vmware, linuxPackages_xanmod_stable.vmware)</li>
    <li>linuxPackages_xanmod_latest.xone (linuxKernel.packages.linux_xanmod_latest.xone, linuxPackages_xanmod_stable.xone)</li>
    <li>linuxPackages_xanmod_latest.xpad-noone (linuxKernel.packages.linux_xanmod_latest.xpad-noone, linuxPackages_xanmod_stable.xpad-noone)</li>
    <li>linuxPackages_xanmod_latest.xpadneo (linuxKernel.packages.linux_xanmod_latest.xpadneo, linuxPackages_xanmod_stable.xpadneo)</li>
    <li>linuxPackages_xanmod_latest.yt6801 (linuxKernel.packages.linux_xanmod_latest.yt6801, linuxPackages_xanmod_stable.yt6801)</li>
    <li>linuxPackages_xanmod_latest.zenergy (linuxKernel.packages.linux_xanmod_latest.zenergy, linuxPackages_xanmod_stable.zenergy)</li>
    <li>linuxPackages_xanmod_latest.zenpower (linuxKernel.packages.linux_xanmod_latest.zenpower, linuxPackages_xanmod_stable.zenpower)</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 124 packages built:</summary>
  <ul>
    <li>linux_xanmod (linuxPackages_xanmod.kernel)</li>
    <li>linux_xanmod.dev (linuxPackages_xanmod.kernel.dev)</li>
    <li>linux_xanmod.modules (linuxPackages_xanmod.kernel.modules)</li>
    <li>linuxPackages_xanmod.acer-wmi-battery (linuxKernel.packages.linux_xanmod.acer-wmi-battery)</li>
    <li>linuxPackages_xanmod.acpi_call (linuxKernel.packages.linux_xanmod.acpi_call)</li>
    <li>linuxPackages_xanmod.ajantv2 (linuxKernel.packages.linux_xanmod.ajantv2)</li>
    <li>linuxPackages_xanmod.akvcam (linuxKernel.packages.linux_xanmod.akvcam)</li>
    <li>linuxPackages_xanmod.amdgpu-i2c (linuxKernel.packages.linux_xanmod.amdgpu-i2c)</li>
    <li>linuxPackages_xanmod.amneziawg (linuxKernel.packages.linux_xanmod.amneziawg)</li>
    <li>linuxPackages_xanmod.apfs (linuxKernel.packages.linux_xanmod.apfs)</li>
    <li>linuxPackages_xanmod.asus-ec-sensors (linuxKernel.packages.linux_xanmod.asus-ec-sensors)</li>
    <li>linuxPackages_xanmod.batman_adv (linuxKernel.packages.linux_xanmod.batman_adv)</li>
    <li>linuxPackages_xanmod.bbswitch (linuxKernel.packages.linux_xanmod.bbswitch)</li>
    <li>linuxPackages_xanmod.bcachefs (linuxKernel.packages.linux_xanmod.bcachefs)</li>
    <li>linuxPackages_xanmod.ch9344 (linuxKernel.packages.linux_xanmod.ch9344)</li>
    <li>linuxPackages_xanmod.chipsec (linuxKernel.packages.linux_xanmod.chipsec)</li>
    <li>linuxPackages_xanmod.chipsec.dist (linuxKernel.packages.linux_xanmod.chipsec.dist)</li>
    <li>linuxPackages_xanmod.corefreq (linuxKernel.packages.linux_xanmod.corefreq)</li>
    <li>linuxPackages_xanmod.cpupower (linuxKernel.packages.linux_xanmod.cpupower)</li>
    <li>linuxPackages_xanmod.cryptodev (linuxKernel.packages.linux_xanmod.cryptodev)</li>
    <li>linuxPackages_xanmod.ddcci-driver (linuxKernel.packages.linux_xanmod.ddcci-driver)</li>
    <li>linuxPackages_xanmod.decklink (linuxKernel.packages.linux_xanmod.decklink)</li>
    <li>linuxPackages_xanmod.digimend (linuxKernel.packages.linux_xanmod.digimend)</li>
    <li>linuxPackages_xanmod.dpdk-kmods (linuxKernel.packages.linux_xanmod.dpdk-kmods)</li>
    <li>linuxPackages_xanmod.drbd (linuxKernel.packages.linux_xanmod.drbd)</li>
    <li>linuxPackages_xanmod.ecapture (linuxKernel.packages.linux_xanmod.ecapture)</li>
    <li>linuxPackages_xanmod.ena (linuxKernel.packages.linux_xanmod.ena)</li>
    <li>linuxPackages_xanmod.evdi (linuxKernel.packages.linux_xanmod.evdi)</li>
    <li>linuxPackages_xanmod.facetimehd (linuxKernel.packages.linux_xanmod.facetimehd)</li>
    <li>linuxPackages_xanmod.fanout (linuxKernel.packages.linux_xanmod.fanout)</li>
    <li>linuxPackages_xanmod.framework-laptop-kmod (linuxKernel.packages.linux_xanmod.framework-laptop-kmod)</li>
    <li>linuxPackages_xanmod.fwts-efi-runtime (linuxKernel.packages.linux_xanmod.fwts-efi-runtime)</li>
    <li>linuxPackages_xanmod.gasket (linuxKernel.packages.linux_xanmod.gasket)</li>
    <li>linuxPackages_xanmod.gcadapter-oc-kmod (linuxKernel.packages.linux_xanmod.gcadapter-oc-kmod)</li>
    <li>linuxPackages_xanmod.hid-fanatecff (linuxKernel.packages.linux_xanmod.hid-fanatecff)</li>
    <li>linuxPackages_xanmod.hid-ite8291r3 (linuxKernel.packages.linux_xanmod.hid-ite8291r3)</li>
    <li>linuxPackages_xanmod.hid-t150 (linuxKernel.packages.linux_xanmod.hid-t150)</li>
    <li>linuxPackages_xanmod.hid-tmff2 (linuxKernel.packages.linux_xanmod.hid-tmff2)</li>
    <li>linuxPackages_xanmod.hpuefi-mod (linuxKernel.packages.linux_xanmod.hpuefi-mod)</li>
    <li>linuxPackages_xanmod.hyperv-daemons (linuxKernel.packages.linux_xanmod.hyperv-daemons)</li>
    <li>linuxPackages_xanmod.hyperv-daemons.bin (linuxKernel.packages.linux_xanmod.hyperv-daemons.bin)</li>
    <li>linuxPackages_xanmod.hyperv-daemons.lib (linuxKernel.packages.linux_xanmod.hyperv-daemons.lib)</li>
    <li>linuxPackages_xanmod.iio-utils (linuxKernel.packages.linux_xanmod.iio-utils)</li>
    <li>linuxPackages_xanmod.ipu6-drivers (linuxKernel.packages.linux_xanmod.ipu6-drivers)</li>
    <li>linuxPackages_xanmod.it87 (linuxKernel.packages.linux_xanmod.it87)</li>
    <li>linuxPackages_xanmod.jool (linuxKernel.packages.linux_xanmod.jool)</li>
    <li>linuxPackages_xanmod.kvmfr (linuxKernel.packages.linux_xanmod.kvmfr)</li>
    <li>linuxPackages_xanmod.lenovo-legion-module (linuxKernel.packages.linux_xanmod.lenovo-legion-module)</li>
    <li>linuxPackages_xanmod.linux-gpib (linuxKernel.packages.linux_xanmod.linux-gpib)</li>
    <li>linuxPackages_xanmod.liquidtux (linuxKernel.packages.linux_xanmod.liquidtux)</li>
    <li>linuxPackages_xanmod.lkrg (linuxKernel.packages.linux_xanmod.lkrg)</li>
    <li>linuxPackages_xanmod.mdio-netlink (linuxKernel.packages.linux_xanmod.mdio-netlink)</li>
    <li>linuxPackages_xanmod.msi-ec (linuxKernel.packages.linux_xanmod.msi-ec)</li>
    <li>linuxPackages_xanmod.mstflint_access (linuxKernel.packages.linux_xanmod.mstflint_access)</li>
    <li>linuxPackages_xanmod.nct6687d (linuxKernel.packages.linux_xanmod.nct6687d)</li>
    <li>linuxPackages_xanmod.netatop (linuxKernel.packages.linux_xanmod.netatop)</li>
    <li>linuxPackages_xanmod.new-lg4ff (linuxKernel.packages.linux_xanmod.new-lg4ff)</li>
    <li>linuxPackages_xanmod.nullfs (linuxKernel.packages.linux_xanmod.nullfs)</li>
    <li>linuxPackages_xanmod.nvidia_x11_beta_open (linuxKernel.packages.linux_xanmod.nvidia_x11_beta_open)</li>
    <li>linuxPackages_xanmod.nvidia_x11_latest_open (linuxKernel.packages.linux_xanmod.nvidia_x11_latest_open, linuxPackages_xanmod.nvidia_x11_production_open, linuxPackages_xanmod.nvidia_x11_stable_open)</li>
    <li>linuxPackages_xanmod.nvidia_x11_vulkan_beta_open (linuxKernel.packages.linux_xanmod.nvidia_x11_vulkan_beta_open)</li>
    <li>linuxPackages_xanmod.nxp-pn5xx (linuxKernel.packages.linux_xanmod.nxp-pn5xx)</li>
    <li>linuxPackages_xanmod.openafs (linuxKernel.packages.linux_xanmod.openafs, linuxPackages_xanmod.openafs_1_8)</li>
    <li>linuxPackages_xanmod.openrazer (linuxKernel.packages.linux_xanmod.openrazer)</li>
    <li>linuxPackages_xanmod.opensnitch-ebpf (linuxKernel.packages.linux_xanmod.opensnitch-ebpf)</li>
    <li>linuxPackages_xanmod.ply (linuxKernel.packages.linux_xanmod.ply)</li>
    <li>linuxPackages_xanmod.qc71_laptop (linuxKernel.packages.linux_xanmod.qc71_laptop)</li>
    <li>linuxPackages_xanmod.r8125 (linuxKernel.packages.linux_xanmod.r8125)</li>
    <li>linuxPackages_xanmod.r8168 (linuxKernel.packages.linux_xanmod.r8168)</li>
    <li>linuxPackages_xanmod.rr-zen_workaround (linuxKernel.packages.linux_xanmod.rr-zen_workaround)</li>
    <li>linuxPackages_xanmod.rtl8189es (linuxKernel.packages.linux_xanmod.rtl8189es)</li>
    <li>linuxPackages_xanmod.rtl8189fs (linuxKernel.packages.linux_xanmod.rtl8189fs)</li>
    <li>linuxPackages_xanmod.rtl8821ce (linuxKernel.packages.linux_xanmod.rtl8821ce)</li>
    <li>linuxPackages_xanmod.rtl8821cu (linuxKernel.packages.linux_xanmod.rtl8821cu)</li>
    <li>linuxPackages_xanmod.rtl88x2bu (linuxKernel.packages.linux_xanmod.rtl88x2bu)</li>
    <li>linuxPackages_xanmod.rtw88 (linuxKernel.packages.linux_xanmod.rtw88)</li>
    <li>linuxPackages_xanmod.rust-out-of-tree-module (linuxKernel.packages.linux_xanmod.rust-out-of-tree-module)</li>
    <li>linuxPackages_xanmod.ryzen-smu (linuxKernel.packages.linux_xanmod.ryzen-smu)</li>
    <li>linuxPackages_xanmod.sheep-net (linuxKernel.packages.linux_xanmod.sheep-net)</li>
    <li>linuxPackages_xanmod.shufflecake (linuxKernel.packages.linux_xanmod.shufflecake)</li>
    <li>linuxPackages_xanmod.shufflecake.bin (linuxKernel.packages.linux_xanmod.shufflecake.bin)</li>
    <li>linuxPackages_xanmod.sysdig (linuxKernel.packages.linux_xanmod.sysdig)</li>
    <li>linuxPackages_xanmod.system76 (linuxKernel.packages.linux_xanmod.system76)</li>
    <li>linuxPackages_xanmod.system76-acpi (linuxKernel.packages.linux_xanmod.system76-acpi)</li>
    <li>linuxPackages_xanmod.system76-io (linuxKernel.packages.linux_xanmod.system76-io)</li>
    <li>linuxPackages_xanmod.systemtap (linuxKernel.packages.linux_xanmod.systemtap)</li>
    <li>linuxPackages_xanmod.tmon (linuxKernel.packages.linux_xanmod.tmon)</li>
    <li>linuxPackages_xanmod.tp_smapi (linuxKernel.packages.linux_xanmod.tp_smapi)</li>
    <li>linuxPackages_xanmod.trelay (linuxKernel.packages.linux_xanmod.trelay)</li>
    <li>linuxPackages_xanmod.tsme-test (linuxKernel.packages.linux_xanmod.tsme-test)</li>
    <li>linuxPackages_xanmod.tt-kmd (linuxKernel.packages.linux_xanmod.tt-kmd)</li>
    <li>linuxPackages_xanmod.turbostat (linuxKernel.packages.linux_xanmod.turbostat)</li>
    <li>linuxPackages_xanmod.tuxedo-drivers (linuxKernel.packages.linux_xanmod.tuxedo-drivers)</li>
    <li>linuxPackages_xanmod.universal-pidff (linuxKernel.packages.linux_xanmod.universal-pidff)</li>
    <li>linuxPackages_xanmod.usbip (linuxKernel.packages.linux_xanmod.usbip)</li>
    <li>linuxPackages_xanmod.v4l2loopback (linuxKernel.packages.linux_xanmod.v4l2loopback)</li>
    <li>linuxPackages_xanmod.v4l2loopback.bin (linuxKernel.packages.linux_xanmod.v4l2loopback.bin)</li>
    <li>linuxPackages_xanmod.v86d (linuxKernel.packages.linux_xanmod.v86d)</li>
    <li>linuxPackages_xanmod.veikk-linux-driver (linuxKernel.packages.linux_xanmod.veikk-linux-driver)</li>
    <li>linuxPackages_xanmod.vendor-reset (linuxKernel.packages.linux_xanmod.vendor-reset)</li>
    <li>linuxPackages_xanmod.vhba (linuxKernel.packages.linux_xanmod.vhba)</li>
    <li>linuxPackages_xanmod.virtio_vmmci (linuxKernel.packages.linux_xanmod.virtio_vmmci)</li>
    <li>linuxPackages_xanmod.virtualbox (linuxKernel.packages.linux_xanmod.virtualbox)</li>
    <li>linuxPackages_xanmod.virtualboxGuestAdditions (linuxKernel.packages.linux_xanmod.virtualboxGuestAdditions)</li>
    <li>linuxPackages_xanmod.vmware (linuxKernel.packages.linux_xanmod.vmware)</li>
    <li>linuxPackages_xanmod.x86_energy_perf_policy (linuxKernel.packages.linux_xanmod.x86_energy_perf_policy)</li>
    <li>linuxPackages_xanmod.xone (linuxKernel.packages.linux_xanmod.xone)</li>
    <li>linuxPackages_xanmod.xpad-noone (linuxKernel.packages.linux_xanmod.xpad-noone)</li>
    <li>linuxPackages_xanmod.xpadneo (linuxKernel.packages.linux_xanmod.xpadneo)</li>
    <li>linuxPackages_xanmod.yt6801 (linuxKernel.packages.linux_xanmod.yt6801)</li>
    <li>linuxPackages_xanmod.zenergy (linuxKernel.packages.linux_xanmod.zenergy)</li>
    <li>linuxPackages_xanmod.zenpower (linuxKernel.packages.linux_xanmod.zenpower)</li>
    <li>linuxPackages_xanmod.zfs_2_3 (linuxKernel.packages.linux_xanmod.zfs_2_3)</li>
    <li>linuxPackages_xanmod.zfs_2_4 (linuxKernel.packages.linux_xanmod.zfs_2_4, linuxPackages_xanmod.zfs_unstable)</li>
    <li>linuxPackages_xanmod_latest.cpupower (linuxKernel.packages.linux_xanmod_latest.cpupower, linuxPackages_xanmod_stable.cpupower)</li>
    <li>linuxPackages_xanmod_latest.hyperv-daemons (linuxKernel.packages.linux_xanmod_latest.hyperv-daemons, linuxPackages_xanmod_stable.hyperv-daemons)</li>
    <li>linuxPackages_xanmod_latest.hyperv-daemons.bin (linuxKernel.packages.linux_xanmod_latest.hyperv-daemons.bin, linuxPackages_xanmod_stable.hyperv-daemons.bin)</li>
    <li>linuxPackages_xanmod_latest.hyperv-daemons.lib (linuxKernel.packages.linux_xanmod_latest.hyperv-daemons.lib, linuxPackages_xanmod_stable.hyperv-daemons.lib)</li>
    <li>linuxPackages_xanmod_latest.iio-utils (linuxKernel.packages.linux_xanmod_latest.iio-utils, linuxPackages_xanmod_stable.iio-utils)</li>
    <li>linuxPackages_xanmod_latest.ply (linuxKernel.packages.linux_xanmod_latest.ply, linuxPackages_xanmod_stable.ply)</li>
    <li>linuxPackages_xanmod_latest.tmon (linuxKernel.packages.linux_xanmod_latest.tmon, linuxPackages_xanmod_stable.tmon)</li>
    <li>linuxPackages_xanmod_latest.turbostat (linuxKernel.packages.linux_xanmod_latest.turbostat, linuxPackages_xanmod_stable.turbostat)</li>
    <li>linuxPackages_xanmod_latest.usbip (linuxKernel.packages.linux_xanmod_latest.usbip, linuxPackages_xanmod_stable.usbip)</li>
    <li>linuxPackages_xanmod_latest.x86_energy_perf_policy (linuxKernel.packages.linux_xanmod_latest.x86_energy_perf_policy, linuxPackages_xanmod_stable.x86_energy_perf_policy)</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>linuxPackages_xanmod.lttng-modules</summary>
<pre>      |         ^~~~~~~~~~~~~~~~~~~~~~~~
probes/../../include/lttng/tracepoint-event-impl.h:838:9: note: in expansion of macro &#x27;_ctf_integer_ext_isuser0&#x27;
  838 |         _ctf_integer_ext_isuser##_user(_type, _item, _user_src, _byte_order, _base, _nowrite)
      |         ^~~~~~~~~~~~~~~~~~~~~~~
probes/../../include/lttng/events-write.h:10:9: note: in expansion of macro &#x27;_ctf_integer_ext&#x27;
   10 |         _ctf_integer_ext(_type, _item, _src, __BYTE_ORDER, 10, 0, 0)
      |         ^~~~~~~~~~~~~~~~
probes/../../include/instrumentation/events/btrfs.h:434:17: note: in expansion of macro &#x27;ctf_integer&#x27;
  434 |                 ctf_integer(int, refs, refcount_read(&amp;ordered-&gt;refs))
      |                 ^~~~~~~~~~~
  CC [M]  probes/lttng-probe-printk.o
make[5]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/scripts/Makefile.build:287: probes/lttng-probe-btrfs.o] Error 1
make[5]: *** Waiting for unfinished jobs....
make[4]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/scripts/Makefile.build:544: probes] Error 2
make[3]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/Makefile:2026: .] Error 2
make[2]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/Makefile:248: __sub-make] Error 2
make[2]: Leaving directory &#x27;/build/source/src&#x27;
make[1]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/Makefile:248: __sub-make] Error 2
make[1]: Leaving directory &#x27;/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/build&#x27;
make: *** [Makefile:31: modules] Error 2</pre>
</details>
<details>
<summary>linuxPackages_xanmod.mbp2018-bridge-drv</summary>
<pre>      |                                   ^~~~~~~~~~~
In file included from /nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/include/linux/device.h:30,
                 from /nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/include/linux/pci.h:37,
                 from vhci/../queue.h:5,
                 from vhci/queue.h:5:
/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/include/linux/device/class.h:226:54: note: expected &#x27;const char *&#x27; but argument is of type &#x27;struct module *&#x27;
  226 | struct class * __must_check class_create(const char *name);
      |                                          ~~~~~~~~~~~~^~~~
vhci/vhci.c:725:22: error: too many arguments to function &#x27;class_create&#x27;; expected 1, have 2
  725 |     bce_vhci_class = class_create(THIS_MODULE, &quot;bce-vhci&quot;);
      |                      ^~~~~~~~~~~~              ~~~~~~~~~~
/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/include/linux/device/class.h:226:29: note: declared here
  226 | struct class * __must_check class_create(const char *name);
      |                             ^~~~~~~~~~~~
make[3]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/scripts/Makefile.build:287: vhci/vhci.o] Error 1
make[2]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/Makefile:2026: .] Error 2
make[1]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/Makefile:248: __sub-make] Error 2
make[1]: Leaving directory &#x27;/build/source&#x27;
make: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/Makefile:248: __sub-make] Error 2
make: Leaving directory &#x27;/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/build&#x27;</pre>
</details>
<details>
<summary>linuxPackages_xanmod.vmm_clock</summary>
<pre>make[2]: Entering directory &#x27;/build/source&#x27;
  CC [M]  vmm_clock_main.o
  CC [M]  pvclock.o
pvclock.c: In function &#x27;pvclock_clocksource_read&#x27;:
pvclock.c:64:17: error: implicit declaration of function &#x27;rdtsc&#x27; [8;;https://gcc.gnu.org/onlinedocs/gcc-15.2.0/gcc/Warning-Options.html#index-Wimplicit-function-declaration-Wimplicit-function-declaration8;;]
   64 |         delta = rdtsc() - tsc_timestamp;
      |                 ^~~~~
make[4]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/scripts/Makefile.build:287: pvclock.o] Error 1
make[4]: *** Waiting for unfinished jobs....
vmm_clock_main.c: In function &#x27;vmm_detect&#x27;:
vmm_clock_main.c:134:24: error: implicit declaration of function &#x27;hypervisor_cpuid_base&#x27; [8;;https://gcc.gnu.org/onlinedocs/gcc-15.2.0/gcc/Warning-Options.html#index-Wimplicit-function-declaration-Wimplicit-function-declaration8;;]
  134 |                 return hypervisor_cpuid_base(&quot;OpenBSDVMM58&quot;, 0);
      |                        ^~~~~~~~~~~~~~~~~~~~~
make[4]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/scripts/Makefile.build:287: vmm_clock_main.o] Error 1
make[3]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/Makefile:2026: .] Error 2
make[2]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/Makefile:248: __sub-make] Error 2
make[2]: Leaving directory &#x27;/build/source&#x27;
make[1]: *** [/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/source/Makefile:248: __sub-make] Error 2
make[1]: Leaving directory &#x27;/nix/store/swb8dciva14xqfci8ywlysxxq7zp7m5s-linux-xanmod-6.18.25-dev/lib/modules/6.18.25-xanmod1/build&#x27;
make: *** [Makefile:14: module] Error 2</pre>
</details>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 16 packages built:</summary>
  <ul>
    <li>linuxPackages_xanmod.cpupower (linuxKernel.packages.linux_xanmod.cpupower)</li>
    <li>linuxPackages_xanmod.hyperv-daemons (linuxKernel.packages.linux_xanmod.hyperv-daemons)</li>
    <li>linuxPackages_xanmod.hyperv-daemons.bin (linuxKernel.packages.linux_xanmod.hyperv-daemons.bin)</li>
    <li>linuxPackages_xanmod.hyperv-daemons.lib (linuxKernel.packages.linux_xanmod.hyperv-daemons.lib)</li>
    <li>linuxPackages_xanmod.iio-utils (linuxKernel.packages.linux_xanmod.iio-utils)</li>
    <li>linuxPackages_xanmod.ply (linuxKernel.packages.linux_xanmod.ply)</li>
    <li>linuxPackages_xanmod.tmon (linuxKernel.packages.linux_xanmod.tmon)</li>
    <li>linuxPackages_xanmod.usbip (linuxKernel.packages.linux_xanmod.usbip)</li>
    <li>linuxPackages_xanmod_latest.cpupower (linuxKernel.packages.linux_xanmod_latest.cpupower, linuxPackages_xanmod_stable.cpupower)</li>
    <li>linuxPackages_xanmod_latest.hyperv-daemons (linuxKernel.packages.linux_xanmod_latest.hyperv-daemons, linuxPackages_xanmod_stable.hyperv-daemons)</li>
    <li>linuxPackages_xanmod_latest.hyperv-daemons.bin (linuxKernel.packages.linux_xanmod_latest.hyperv-daemons.bin, linuxPackages_xanmod_stable.hyperv-daemons.bin)</li>
    <li>linuxPackages_xanmod_latest.hyperv-daemons.lib (linuxKernel.packages.linux_xanmod_latest.hyperv-daemons.lib, linuxPackages_xanmod_stable.hyperv-daemons.lib)</li>
    <li>linuxPackages_xanmod_latest.iio-utils (linuxKernel.packages.linux_xanmod_latest.iio-utils, linuxPackages_xanmod_stable.iio-utils)</li>
    <li>linuxPackages_xanmod_latest.ply (linuxKernel.packages.linux_xanmod_latest.ply, linuxPackages_xanmod_stable.ply)</li>
    <li>linuxPackages_xanmod_latest.tmon (linuxKernel.packages.linux_xanmod_latest.tmon, linuxPackages_xanmod_stable.tmon)</li>
    <li>linuxPackages_xanmod_latest.usbip (linuxKernel.packages.linux_xanmod_latest.usbip, linuxPackages_xanmod_stable.usbip)</li>
  </ul>
</details>

--- a/pkgs/os-specific/linux/kernel/xanmod-kernels.nix
+++ b/pkgs/os-specific/linux/kernel/xanmod-kernels.nix
@@ -21,8 +21,8 @@ let
     };
     # ./update-xanmod.sh main
     main = {
-      version = "6.19.14";
-      hash = "sha256-xp5IBA4epb+3SCJIkbxqzymwvz4T9p7NOP9y6iy6DZk=";
+      version = "7.0.2";
+      hash = "sha256-RXdgP6jpy6GfZ70WG6k6rrryjHV2uUaqy4e+4jCgJek=";
     };
   };
 
@@ -58,7 +58,6 @@ let
 
             # Preemption
             PREEMPT = lib.mkOverride 60 yes;
-            PREEMPT_VOLUNTARY = lib.mkOverride 60 no;
 
             # Google's BBRv3 TCP congestion Control
             TCP_CONG_BBR = yes;
@@ -89,6 +88,9 @@ let
             # x86 features
             X86_FRED = yes;
             X86_POSTED_MSI = yes;
+          }
+          // lib.optionalAttrs (lib.versionOlder (lib.versions.majorMinor version) "7.0") {
+            PREEMPT_VOLUNTARY = lib.mkOverride 60 no;
           }
           // lib.optionalAttrs (lib.versionAtLeast (lib.versions.majorMinor version) "6.13") {
             # Lazy preemption

